### DISCUSSION TITLE
Option to disable the Backpack from being equipped on Chestplate slot

### DISCUSSION THREAD

**Discussion by MatsuKari** [P3pp3rF1y/SophisticatedBackpacks#1057] [Ideas]
Add an option in the configs of the mod to disable Backpacks from being equipped in Chestplate slot, being able to be equipped only on curios:back slot

**Comment by P3pp3rF1y** [comment]
Could you explain to me why this is needed? I mean it's just an option for player to be able to put it there no one is forcing them and it's clearly a disadvantage because you lose an armor so I wonder why we need to force player to not be able to use the slot

**Comment by MatsuKari** [comment]
i mean, the more options the player has the better it is, isn't it? also, why would i use the Backpack in my chestplate slot, if i have a better armor to use instead? Why would i use a Backpack that's meant for item storage as a armor/chestplate? The Backpack should be used to store your items and not occupy a important slot in the Player's Inventory like the Chestplate inventory. But regardless of that, the reason why i asked for such feature is because i slotted a couple gems from apotheosis on my backpack, when it's on my chest it applies the buffs from the gems, but when in curios:back it doesn't. When i talked to the author of Apothic Curios, they said it's because the item also counts as a Chestplate, that's why i want to disable it from being eqquipable on Chest.

**Comment by MatsuKari** [comment]
![Screenshot_4](https://github.com/P3pp3rF1y/SophisticatedBackpacks/assets/85963049/f1438f70-4c6c-4ede-a679-386e47a98e81)
- First, there is this Warlord Gem from Apotheosis, which is the gem i slotted in my backpack for example purposes. Note that it has different buffs for each place it can be equipped on player. The one i wanted is the +65% Max Health when the backpack is equipped on Curios Back.
![Screenshot_6](https://github.com/P3pp3rF1y/SophisticatedBackpacks/assets/85963049/91ddcb8a-381e-4b4e-84d1-ffffbdd30434)
- Note that this Charm equipped on the Charm Slot does have the correct buff application for it
![Screenshot_5](https://github.com/P3pp3rF1y/SophisticatedBackpacks/assets/85963049/9f533ccd-7e57-4014-b29f-440a186fc43c)
- Note that the Backpack does have the buff stat for Chestplate and not for the Curios:Back slot 
![Screenshot_7](https://github.com/P3pp3rF1y/SophisticatedBackpacks/assets/85963049/d1ed4335-3324-4e9d-9626-79bbc0a443ab)
- Note that when it's equipped on the Chestplate it does apply the effect of the gems
![Screenshot_8](https://github.com/P3pp3rF1y/SophisticatedBackpacks/assets/85963049/8e9e03b5-5809-4f76-a021-b439956d91d3)
- Finally, when it's on curios:back it does not.

**Comment by MatsuKari** [comment]
Why use the chestplate slot for the backpack if it could just be equipped in the curios
slot? Why give up your actual chestplate/armor for the backpack if it's just an accessory and could be equipped somewhere else? Remember that there are mods, like Ad Astra, where you need to have all parts of your armor equipped to get its effects. In the case of Ad Astra, you die from lack of oxygen in space if a single piece of the armor isn't equipped

**Comment by P3pp3rF1y** [comment]
No one is forcing players to use chestplate for their backpack so that argument doesn't really apply here. The reason backpack can go in chestplate slot is so that player can wear it when there's no curios around. I can potentially look into disabling that when curios is in the pack but honestly this is very low on my list of priorities so it may not happen.

**Reply by MatsuKari** [reply]
Since no one is forcing players to use the chestplate slot for their backpack, then tell me, as a developer and creator: Why would YOU switch your armor for a backpack in places like The End, the Nether, or other dangerous areas, risking having no defense against potentially dangerous mobs? I know Curios is an optional dependency, but wouldn't it be more beneficial to make it a required dependency and leave the backpack only as an accessory? Or did you not understand the point here, even with the Ad Astra example given?

**Reply by P3pp3rF1y** [reply]
I don't think that I am forcing players to wear it in chestplate it's up to them so I don't understand the point here. And I am definitely not making curios required depending on that's for sure.

**Comment by edralzar** [comment]
@P3pp3rF1y one practical case this could be useful for is to avoid equipping / switching a backpack upon use from the hotbar. 

I'd very much like to open the backpack in such a case rather than replacing my armor in the chest slot when right clicking the slot.

perhaps there's a vanilla / 3rd party mod that would help with that 🤔

**Reply by P3pp3rF1y** [reply]
Backpack doesn't get placed in chest slot when you simply right click it. And as far as I can see it doesn't do that even with shift click. Instead in both cases it just opens.
So that's another mod being "smart" and putting it in chest slot.

**Reply by edralzar** [reply]
ah no sorry I meant with the backpack slot selected in the hotbat and right clicking into the air, not the slot... but that might also be a mod interfering I guesd

**Reply by P3pp3rF1y** [reply]
yes that's exactly what I did, right click air with backpack in hand, also tried with shift right click. So it's another mod doing this swapping for you.
